### PR TITLE
Update README for SDK Manager

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,34 @@ configuration.setVersion("v1");
 // deploy
 deployment.deploy(deployConfiguration);
 ```
+
+## SDK Manager/Downloader
```
Comment (etanshaul) [inline README.md]: can you just call it the SDK Manager? downloading is part of what the manager does. I don't really care though.

Comment (loosebazooka) [reply README.md]: yeah

**Comment by loosebazooka** [comment]
Hrmm, tests broken on windows? I'm taking a look.

File: README.md [hunk 2]
```diff
@@ -57,3 +57,34 @@ configuration.setVersion("v1");
 // deploy
 deployment.deploy(deployConfiguration);
 ```
+
+## SDK Manager
+
+When using the Cloud SDK manager to provide a cloud SDK to users, the implementer must manage all operations. The operations are intended to run asynchronously, either on an executor or through mechanisms provided by an IDE.
```
Comment (elharo) [inline README.md]: I don't know what you mean by "must manage all operations."

File: README.md [hunk 3]
```diff
@@ -57,3 +57,34 @@ configuration.setVersion("v1");
...
+
+```java
+// Create a new Managed SDK instance
+ManagedCloudSdk sdk = ManagedCloudSdk.newManagedSdk("123.123.123") // SDK fixed at version.
+ManagedCloudSdk sdk = ManagedCloudSdk.newManagedSdk() // 'LATEST' sdk, can be updated.
+
+// Implement the listener interface to listen to operation output
+MessageListener listener = new MessageListener() {...};
+
+// Always check if operations are needed before running them
```
Comment (elharo) [inline README.md]: If we always have to do this, we should push this into the install command itself. Just make it a noop if the SDK is already installed.

File: README.md (hunk 2 above)
Comment (loosebazooka) [reply README.md]: Yeah, I guess what I mean is that it's not updated automatically... I'll change this text.

File: README.md (hunk 3 above)
Comment (loosebazooka) [reply README.md]: Install's behavior is to write over a previous install. Although I don't see a library user using this in more than one place in their code, they could potentially wrap it on their own.

Comment (elharo) [reply README.md]: Again, if users really have to do this on every install it's safer and easier to just make that part of the call. However that doesn't need to block this PR.

**Pull request merged**